diff -urN plasma-nm-0.9.3.2/editor/kde-nm-connection-editor.desktop plasma-nm-0.9.3.2-patched/editor/kde-nm-connection-editor.desktop
--- plasma-nm-0.9.3.2/editor/kde-nm-connection-editor.desktop	2013-10-14 14:20:49.000000000 +0400
+++ plasma-nm-0.9.3.2-patched/editor/kde-nm-connection-editor.desktop	2013-10-15 16:40:34.889109727 +0400
@@ -15,6 +15,7 @@
 Name[pt]=kde-nm-connection-editor
 Name[pt_BR]=kde-nm-connection-editor
 Name[ro]=kde-nm-connection-editor
+Name[ru]=kde-nm-connection-editor
 Name[sk]=kde-nm-connection-editor
 Name[sv]=kde-nm-connection-editor
 Name[tr]=kde-nm-connection-editor
@@ -35,6 +36,7 @@
 GenericName[pt]=Editor de ligações
 GenericName[pt_BR]=Editor de conexão
 GenericName[ro]=Redactor de conexiuni
+GenericName[ru]=Редактор сетевых соединений
 GenericName[sk]=Editor pripojení
 GenericName[sv]=Anslutningseditor
 GenericName[tr]=Bağlantı düzenleyici
@@ -56,6 +58,7 @@
 Comment[pt]=Edite as suas ligações de rede
 Comment[pt_BR]=Edite as suas conexões de rede
 Comment[ro]=Redactați-vă conexiunile de rețea
+Comment[ru]=Редактор сетевых соединений
 Comment[sk]=Upraviť vaše sieťové pripojenia
 Comment[sv]=Redigera nätverksanslutningar
 Comment[tr]=Ağ bağlantılarınızı düzenleyin
diff -urN plasma-nm-0.9.3.2/kded/networkmanagement.desktop plasma-nm-0.9.3.2-patched/kded/networkmanagement.desktop
--- plasma-nm-0.9.3.2/kded/networkmanagement.desktop	2013-10-14 14:20:49.000000000 +0400
+++ plasma-nm-0.9.3.2-patched/kded/networkmanagement.desktop	2013-10-15 16:45:29.767423240 +0400
@@ -10,6 +10,7 @@
 Name[pt]=Módulo de Gestão de Rede do Plasma
 Name[pt_BR]=Módulo do Networkmanagement para o Plasma
 Name[ro]=Modul Plasma pentru gestiunea rețelei
+Name[ru]=Модуль plasma networkmanagement
 Name[sk]=Modul Plasma správa siete
 Name[sv]=Plasma nätverkshanteringsmodul
 Name[tr]=Plasma Ağ yönetimi modülü
@@ -29,6 +30,7 @@
 Comment[pt]=Fornece as senhas do serviço NetworkManager
 Comment[pt_BR]=Fornece as senhas do serviço NetworkManager
 Comment[ro]=Furnizează secrete către demonul NetworkManager
+Comment[ru]=Предоставляет пароли демону NetworkManager
 Comment[sk]=Poskytuje tajnosti k démonu NetworkManager
 Comment[sv]=Tillhandahåller hemligheter till nätverkshanteringsdemonen
 Comment[tr]=AğYönetici yardımcısına şifreleri sağlar
diff -urN plasma-nm-0.9.3.2/kded/networkmanagement.notifyrc plasma-nm-0.9.3.2-patched/kded/networkmanagement.notifyrc
--- plasma-nm-0.9.3.2/kded/networkmanagement.notifyrc	2013-10-14 14:20:49.000000000 +0400
+++ plasma-nm-0.9.3.2-patched/kded/networkmanagement.notifyrc	2013-10-15 17:40:20.572282518 +0400
@@ -11,6 +11,7 @@
 Name[pt]=Gestão de Rede
 Name[pt_BR]=networkmanagement
 Name[ro]=networkmanagement
+Name[ru]=Управление сетевыми соединениями
 Name[sk]=správa siete
 Name[sv]=nätverkshantering
 Name[tr]=ağyönetimi
@@ -31,6 +32,7 @@
 Comment[pt]=Notificações de erros na rede
 Comment[pt_BR]=Notificações de erros na rede
 Comment[ro]=Notifică despre erorile de rețea
+Comment[ru]=Уведомления об ошибках сетевых соединений
 Comment[sk]=Upozorní na sieťové chyby
 Comment[sv]=Underrättar om nätverksfel
 Comment[tr]=Ağ hatalarını bildirir
@@ -52,6 +54,7 @@
 Name[pt]=Não foi possível activar o dispositivo
 Name[pt_BR]=Não foi possível ativar o dispositivo
 Name[ro]=Activarea dispozitivului a eșuat
+Name[ru]=Не удалось активировать устройство
 Name[sk]=Zlyhala aktivácia zariadenia
 Name[sv]=Aktivering av enheten misslyckades
 Name[tr]=Aygıt etkinleştirilemedi
@@ -73,6 +76,7 @@
 Name[pt]=O estado da ligação activa mudou
 Name[pt_BR]=O estado da conexão ativa foi alterado
 Name[ro]=Starea conexiunii active s-a schimbat
+Name[ru]=Изменение состояния активного соединения
 Name[sk]=Aktívny stav pripojenia zmenený
 Name[sv]=Aktiv anslutningsstatus ändrad
 Name[tr]=Etkin bağlantı durumu değişti
@@ -94,6 +98,7 @@
 Name[pt]=O estado da ligação VPN mudou
 Name[pt_BR]=O estado da conexão VPN foi alterado
 Name[ro]=Starea conexiunii VPN s-a schimbat
+Name[ru]=Изменение состояния VPN соединения
 Name[sk]=Stav pripojenia VPN zmenený
 Name[sv]=VPN-anslutningsstatus ändrad
 Name[tr]=VPN bağlantısı durumu değişti
diff -urN plasma-nm-0.9.3.2/lib/plasma-networkmanagement-vpnuiplugin.desktop plasma-nm-0.9.3.2-patched/lib/plasma-networkmanagement-vpnuiplugin.desktop
--- plasma-nm-0.9.3.2/lib/plasma-networkmanagement-vpnuiplugin.desktop	2013-10-14 14:20:49.000000000 +0400
+++ plasma-nm-0.9.3.2-patched/lib/plasma-networkmanagement-vpnuiplugin.desktop	2013-10-15 16:49:38.131318264 +0400
@@ -14,6 +14,7 @@
 Name[pt]='Plugin' de Interface VPN da Gestão de Rede
 Name[pt_BR]=Plugin de interface VPN do gerenciamento de redes
 Name[ro]=Modul de interfață VPN pentru gestiunea rețelei
+Name[ru]=Графический модуль для настройки VPN
 Name[sk]=Modul pre správu siete VPN
 Name[sv]=Insticksprogram till nätverkshantering med VPN-användargränssnitt
 Name[tr]=Ağ Yönetimi VPN UI Eklentisi
@@ -35,6 +36,7 @@
 Comment[pt]=Define os 'plugins' de interface da Gestão de Rede
 Comment[pt_BR]=Define os plugins de interface do gerenciamento de redes
 Comment[ro]=Definește modulele de interfață pentru gestiunea rețelei
+Comment[ru]=Назначает модули для управления сетями в KDE
 Comment[sk]=Definuje moduly pre správu siete KDE
 Comment[sv]=Definierar insticksprogram för KDE:s nätverkshantering
 Comment[tr]=KDE Ağ Yönetimi UI Eklentilerini tanımlar
diff -urN plasma-nm-0.9.3.2/settings/notifications/networkmanagement_notifications.desktop plasma-nm-0.9.3.2-patched/settings/notifications/networkmanagement_notifications.desktop
--- plasma-nm-0.9.3.2/settings/notifications/networkmanagement_notifications.desktop	2013-10-14 14:20:49.000000000 +0400
+++ plasma-nm-0.9.3.2-patched/settings/notifications/networkmanagement_notifications.desktop	2013-10-15 16:51:38.573812526 +0400
@@ -13,6 +13,7 @@
 Name[pt]=Notificações
 Name[pt_BR]=Notificações
 Name[ro]=Notificări
+Name[ru]=Уведомления
 Name[sk]=Upozornenia
 Name[sv]=Underrättelser
 Name[tr]=Bildirimler
@@ -33,6 +34,7 @@
 Comment[pt]=Configurar as notificações da Gestão da Rede
 Comment[pt_BR]=Configura as notificações do gerenciamento de redes
 Comment[ro]=Configurează notificările gestiunii de rețea
+Comment[ru]=Настройка сетевых уведомлений
 Comment[sk]=Nastaviť upozornenia správy siete
 Comment[sv]=Anpassa underrättelser för nätverkshantering
 Comment[tr]=Ağ Yönetim bilgilendirmelerini yapılandır
diff -urN plasma-nm-0.9.3.2/vpn/l2tp/plasmanetworkmanagement_l2tpui.desktop plasma-nm-0.9.3.2-patched/vpn/l2tp/plasmanetworkmanagement_l2tpui.desktop
--- plasma-nm-0.9.3.2/vpn/l2tp/plasmanetworkmanagement_l2tpui.desktop	2013-10-14 14:20:49.000000000 +0400
+++ plasma-nm-0.9.3.2-patched/vpn/l2tp/plasmanetworkmanagement_l2tpui.desktop	2013-10-15 16:53:04.638736566 +0400
@@ -27,6 +27,7 @@
 Name[pt]=L2TP
 Name[pt_BR]=L2TP
 Name[ro]=L2TP
+Name[ru]=L2TP
 Name[sk]=L2TP
 Name[sv]=L2TP
 Name[tr]=L2TP
@@ -43,6 +44,7 @@
 Comment[fr]=Module externe L2TP
 Comment[hu]=L2TP bővítmény
 Comment[nl]=L2TP-plug-in
+Comment[ru]=Модуль L2TP
 Comment[pl]=Wtyczka L2TP
 Comment[pt]='Plugin' do L2TP
 Comment[pt_BR]=Plugin do L2TP
diff -urN plasma-nm-0.9.3.2/vpn/openconnect/plasmanetworkmanagement_openconnectui.desktop plasma-nm-0.9.3.2-patched/vpn/openconnect/plasmanetworkmanagement_openconnectui.desktop
--- plasma-nm-0.9.3.2/vpn/openconnect/plasmanetworkmanagement_openconnectui.desktop	2013-10-14 14:20:49.000000000 +0400
+++ plasma-nm-0.9.3.2-patched/vpn/openconnect/plasmanetworkmanagement_openconnectui.desktop	2013-10-15 16:55:54.961607237 +0400
@@ -27,6 +27,7 @@
 Name[pt]=OpenConnect
 Name[pt_BR]=OpenConnect
 Name[ro]=OpenConnect
+Name[ru]=OpenConnect
 Name[sk]=OpenConnect
 Name[sv]=OpenConnect
 Name[tr]=OpenConnect
@@ -48,6 +49,7 @@
 Comment[pt]='Plugin' do OpenConnect
 Comment[pt_BR]=Plugin do OpenConnect
 Comment[ro]=Extensie OpenConnect
+Comment[ru]=Модуль OpenConnect
 Comment[sk]=Plugin OpenConnect
 Comment[sv]=Insticksprogram för OpenConnect
 Comment[tr]=OpenConnect Eklentisi
diff -urN plasma-nm-0.9.3.2/vpn/openswan/plasmanetworkmanagement_openswanui.desktop plasma-nm-0.9.3.2-patched/vpn/openswan/plasmanetworkmanagement_openswanui.desktop
--- plasma-nm-0.9.3.2/vpn/openswan/plasmanetworkmanagement_openswanui.desktop	2013-10-14 14:20:49.000000000 +0400
+++ plasma-nm-0.9.3.2-patched/vpn/openswan/plasmanetworkmanagement_openswanui.desktop	2013-10-15 16:59:19.500050155 +0400
@@ -27,6 +27,7 @@
 Name[pt]=OpenSwan
 Name[pt_BR]=OpenSwan
 Name[ro]=OpenSwan
+Name[ru]=OpenSwan
 Name[sk]=OpenSwan
 Name[sv]=OpenSwan
 Name[tr]=OpenSwan
@@ -46,6 +47,7 @@
 Comment[pt]='Plugin' do OpenSwan
 Comment[pt_BR]=Plugin do OpenSwan
 Comment[ro]=Extensie OpenSwan
+Comment[ru]=Модуль для OpenSwan
 Comment[sk]=Plugin OpenSwan
 Comment[sv]=Insticksprogram för OpenSwan
 Comment[tr]=OpenSwan Eklentisi
diff -urN plasma-nm-0.9.3.2/vpn/openvpn/plasmanetworkmanagement_openvpnui.desktop plasma-nm-0.9.3.2-patched/vpn/openvpn/plasmanetworkmanagement_openvpnui.desktop
--- plasma-nm-0.9.3.2/vpn/openvpn/plasmanetworkmanagement_openvpnui.desktop	2013-10-14 14:20:49.000000000 +0400
+++ plasma-nm-0.9.3.2-patched/vpn/openvpn/plasmanetworkmanagement_openvpnui.desktop	2013-10-15 17:00:28.921182271 +0400
@@ -27,6 +27,7 @@
 Name[pt]=OpenVPN
 Name[pt_BR]=OpenVPN
 Name[ro]=OpenVPN
+Name[ru]=OpenVPN
 Name[sk]=OpenVPN
 Name[sv]=OpenVPN
 Name[tr]=OpenVPN
@@ -48,6 +49,7 @@
 Comment[pt]='Plugin' do OpenVPN
 Comment[pt_BR]=Plugin do OpenVPN
 Comment[ro]=Extensie OpenVPN
+Comment[ru]=Модуль для OpenVPN
 Comment[sk]=OpenVPN plugin
 Comment[sv]=Insticksprogram för OpenVPN
 Comment[tr]=OpenVPN eklentisi
diff -urN plasma-nm-0.9.3.2/vpn/pptp/plasmanetworkmanagement_pptpui.desktop plasma-nm-0.9.3.2-patched/vpn/pptp/plasmanetworkmanagement_pptpui.desktop
--- plasma-nm-0.9.3.2/vpn/pptp/plasmanetworkmanagement_pptpui.desktop	2013-10-14 14:20:49.000000000 +0400
+++ plasma-nm-0.9.3.2-patched/vpn/pptp/plasmanetworkmanagement_pptpui.desktop	2013-10-15 17:10:45.269476857 +0400
@@ -27,6 +27,7 @@
 Name[pt]=PPTP
 Name[pt_BR]=PPTP
 Name[ro]=PPTP
+Name[ru]=PPTP
 Name[sk]=PPTP
 Name[sv]=PPTP
 Name[tr]=PPTP
@@ -48,6 +49,7 @@
 Comment[pt]='Plugin' de PPTP
 Comment[pt_BR]=Plugin de PPTP
 Comment[ro]=Modul PPTP
+Comment[ru]=Модуль для PPTP
 Comment[sk]=PPTP modul
 Comment[sv]=Insticksprogram för PPTP
 Comment[tr]=PPTP Eklentisi
diff -urN plasma-nm-0.9.3.2/vpn/strongswan/plasmanetworkmanagement_strongswanui.desktop plasma-nm-0.9.3.2-patched/vpn/strongswan/plasmanetworkmanagement_strongswanui.desktop
--- plasma-nm-0.9.3.2/vpn/strongswan/plasmanetworkmanagement_strongswanui.desktop	2013-10-14 14:20:49.000000000 +0400
+++ plasma-nm-0.9.3.2-patched/vpn/strongswan/plasmanetworkmanagement_strongswanui.desktop	2013-10-15 17:13:35.330350804 +0400
@@ -24,6 +24,7 @@
 Name[pt]=StrongSwan
 Name[pt_BR]=StrongSwan
 Name[ro]=StrongSwan
+Name[ru]=StrongSwan
 Name[sk]=StrongSwan
 Name[sv]=StrongSwan
 Name[tr]=StrongSwan
@@ -41,6 +42,7 @@
 Comment[pt]='Plugin' do StrongSwan
 Comment[pt_BR]=Plugin do StrongSwan
 Comment[ro]=Modul StrongSwan
+Comment[ru]=Модуль для StrongSwan
 Comment[sk]=StrongSwan modul
 Comment[sv]=Insticksprogram för StrongSwan
 Comment[tr]=StrongSwan Eklentisi
diff -urN plasma-nm-0.9.3.2/vpn/vpnc/plasmanetworkmanagement_vpncui.desktop plasma-nm-0.9.3.2-patched/vpn/vpnc/plasmanetworkmanagement_vpncui.desktop
--- plasma-nm-0.9.3.2/vpn/vpnc/plasmanetworkmanagement_vpncui.desktop	2013-10-14 14:20:49.000000000 +0400
+++ plasma-nm-0.9.3.2-patched/vpn/vpnc/plasmanetworkmanagement_vpncui.desktop	2013-10-15 17:12:09.050429451 +0400
@@ -27,6 +27,7 @@
 Name[pt]=VPNC
 Name[pt_BR]=VPNC
 Name[ro]=VPNC
+Name[ru]=VPNC
 Name[sk]=VPNC
 Name[sv]=VPNC
 Name[tr]=VPNC
@@ -48,6 +49,7 @@
 Comment[pt]='Plugin' do VPNC
 Comment[pt_BR]=Plugin do VPNC
 Comment[ro]=Modul VPNC
+Comment[ru]=Модуль для VPNC
 Comment[sk]=VPNC modul
 Comment[sv]=Insticksprogram för VPNC
 Comment[tr]=VPNC Eklentisi
